akka-actor/src/main/scala/akka/actor/CoordinatedShutdown.scala

```diff
@@ -353,7 +355,7 @@ final class CoordinatedShutdown private[akka] (
   import CoordinatedShutdown.{ Reason, UnknownReason }
 
   /** INTERNAL API */
-  private[akka] val log = Logging(system, getClass)
+  private[akka] val log = Logging(system, Logging.simpleName(this))
```

Comment:
we wan't the fqcn in loggers. why doesn't getClass work? looks like an ordinary class
try `classOf[CoordinatedShutdown]`

Reply:
I don't know, and I haven't digged why `getClass` doesn't work, will try to use `classOf`